### PR TITLE
Activating Open Collective

### PR DESCRIPTION
**Pull request by xdamman**
This pull request adds backers and sponsors from your Open Collective https://opencollective.com/ifme ❤️
  
  It adds two badges at the top to show the latest number of backers and sponsors. It also adds placeholders so that the avatar/logo of new backers/sponsors can automatically be shown without having to update your README.md. [[more info](https://github.com/opencollective/opencollective/wiki/Github-banner)]. See how it looks on [this repo](https://github.com/apex/apex#backers).
You can also add a "Donate" button to your website and automatically show your backers and sponsors there with our widgets. Have a look here: https://opencollective.com/widgets

  P.S: As with any pull request, feel free to comment or suggest changes. The only thing "required" are the placeholders on the README because we believe it's important to acknowledge the people in your community that are contributing (financially or with code!).

  Thank you for your great contribution to the open source community. You are awesome! 🙌
  And welcome to the open collective community! 😊

  Come chat with us in the #opensource channel on https://slack.opencollective.com - great place to ask questions and share best practices with other open source sustainers!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -55,3 +55,41 @@ Please join our `#socialmedia` channel on Slack! We post amazing content all ove
...
+We also welcome financial contributions in full transparency on our [open collective](https://opencollective.com/ifme).
+Anyone can file an expense. If the expense makes sense for the development of the community, it will be "merged" in the ledger of our open collective by the core contributors and the person who filed the expense will be reimbursed.
+
+
+## Credits
+
+
+### Contributors
+
+Thank you to all the people who have already contributed to ifme!
```
Comment (julianguyen) [inline CONTRIBUTING.md]: Can you spell ifme as if me?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -55,3 +55,41 @@ Please join our `#socialmedia` channel on Slack! We post amazing content all ove
 [Bee Martinez](https://twitter.com/thebeeletter) is our social media manager. She curates all over our social media channels and will work with you to create your content!
 
 As well as new and original content, we also accept content that has already been published.
+
```
Comment (julianguyen) [inline CONTRIBUTING.md]: Can you get rid of this extra empty line?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -55,3 +55,41 @@ Please join our `#socialmedia` channel on Slack! We post amazing content all ove
...
+
+## Financial contributions
+
+We also welcome financial contributions in full transparency on our [open collective](https://opencollective.com/ifme).
+Anyone can file an expense. If the expense makes sense for the development of the community, it will be "merged" in the ledger of our open collective by the core contributors and the person who filed the expense will be reimbursed.
+
+
```
Comment (julianguyen) [inline CONTRIBUTING.md]: Can you get rid of this extra line?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -55,3 +55,41 @@ Please join our `#socialmedia` channel on Slack! We post amazing content all ove
...
+## Credits
+
+
```
Comment (julianguyen) [inline CONTRIBUTING.md]: Can you get rid of this extra line?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -55,3 +55,41 @@ Please join our `#socialmedia` channel on Slack! We post amazing content all ove
...
+### Contributors
+
+Thank you to all the people who have already contributed to ifme!
+<a href="graphs/contributors"><img src="https://opencollective.com/ifme/contributors.svg?width=890" /></a>
+
+
+### Backers
+
+Thank you to all our backers! [[Become a backer](https://opencollective.com/ifme#backer)]
```
Comment (julianguyen) [inline CONTRIBUTING.md]: Can we remove the extra square brackets round this link?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -55,3 +55,41 @@ Please join our `#socialmedia` channel on Slack! We post amazing content all ove
...
+
+
+## Credits
+
+
+### Contributors
+
+Thank you to all the people who have already contributed to ifme!
+<a href="graphs/contributors"><img src="https://opencollective.com/ifme/contributors.svg?width=890" /></a>
+
```
Comment (julianguyen) [inline CONTRIBUTING.md]: Can you get rid of this extra line?

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -55,3 +55,41 @@ Please join our `#socialmedia` channel on Slack! We post amazing content all ove
...
+
+### Backers
+
+Thank you to all our backers! [[Become a backer](https://opencollective.com/ifme#backer)]
+
+<a href="https://opencollective.com/ifme#backers" target="_blank"><img src="https://opencollective.com/ifme/backers.svg?width=890"></a>
+
```
Comment (julianguyen) [inline CONTRIBUTING.md]: Can you get rid of this extra line?

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -55,3 +55,41 @@ Please join our `#socialmedia` channel on Slack! We post amazing content all ove
...
+
+### Sponsors
+
+Thank you to all our sponsors! (please ask your company to also support this open source project by [becoming a sponsor](https://opencollective.com/ifme#sponsor))
+
```
Comment (julianguyen) [inline CONTRIBUTING.md]: Can put a new line here so that the text and images don't look cramped together?

File: README.md [hunk 9]
```diff
@@ -377,6 +377,36 @@ If you create an issue, please tag it with the appropriate label. We use `enhanc
 
 For bugs, please list the reproduction steps and specify if the bug was produced locally or on production. Please also mention what OS and browser you are using.
 
+## Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute]](CONTRIBUTING.md).
```
Comment (julianguyen) [inline README.md]: Can we remove the square brackets around the link?

File: README.md [hunk 10]
```diff
@@ -377,6 +377,36 @@ If you create an issue, please tag it with the appropriate label. We use `enhanc
...
+<a href="graphs/contributors"><img src="https://opencollective.com/ifme/contributors.svg?width=890" /></a>
+
+
+## Backers
+
+Thank you to all our backers! 🙏 [[Become a backer](https://opencollective.com/ifme#backer)]
```
Comment (julianguyen) [inline README.md]: Can we remove the square brackets here too?

File: README.md [hunk 11]
```diff
@@ -377,6 +377,36 @@ If you create an issue, please tag it with the appropriate label. We use `enhanc
...
+
+<a href="https://opencollective.com/ifme#backers" target="_blank"><img src="https://opencollective.com/ifme/backers.svg?width=890"></a>
+
```
Comment (julianguyen) [inline README.md]: Can you get rid of this extra line?

File: README.md [hunk 12]
```diff
@@ -377,6 +377,36 @@ If you create an issue, please tag it with the appropriate label. We use `enhanc
...
+
+## Sponsors
+
+Support this project by becoming a sponsor. Your logo will show up here with a link to your website. [[Become a sponsor](https://opencollective.com/ifme#sponsor)]
+
```
Comment (julianguyen) [inline README.md]: Can we have a new line here so that the text and images don't look cramped?

File: README.md [hunk 13]
```diff
@@ -377,6 +377,36 @@ If you create an issue, please tag it with the appropriate label. We use `enhanc
 
 For bugs, please list the reproduction steps and specify if the bug was produced locally or on production. Please also mention what OS and browser you are using.
 
+## Contributors
```
Comment (julianguyen) [inline README.md]: Can we use `#`?

File: README.md [hunk 14]
```diff
@@ -377,6 +377,36 @@ If you create an issue, please tag it with the appropriate label. We use `enhanc
...
+
+This project exists thanks to all the people who contribute. [[Contribute]](CONTRIBUTING.md).
+<a href="graphs/contributors"><img src="https://opencollective.com/ifme/contributors.svg?width=890" /></a>
+
+
+## Backers
```
Comment (julianguyen) [inline README.md]: Let's use `#` here too

File: README.md [hunk 15]
```diff
@@ -377,6 +377,36 @@ If you create an issue, please tag it with the appropriate label. We use `enhanc
...
+
+Thank you to all our backers! 🙏 [[Become a backer](https://opencollective.com/ifme#backer)]
+
+<a href="https://opencollective.com/ifme#backers" target="_blank"><img src="https://opencollective.com/ifme/backers.svg?width=890"></a>
+
+
+## Sponsors
```
Comment (julianguyen) [inline README.md]: Let's use `#` here too.

File: README.md [hunk 16]
```diff
@@ -1,5 +1,5 @@
 [![CircleCI](https://circleci.com/gh/julianguyen/ifme/tree/master.svg?style=svg)](https://circleci.com/gh/julianguyen/ifme/tree/master)
```
Comment (julianguyen) [inline README.md]: For the headings, Contributors, Backers, and Sponsors let's add that to the Table of Contents

**Comment by xdamman** [comment]
This PR is there as a template, feel free to edit it to match your personal style :)

**Pull request merged**